nyc_taxi/anaconda-project.yml

```diff
@@ -14,26 +14,26 @@ examples_config:
 user_fields: [examples_config]
 
 channels:
-- pyviz
+- pyviz/label/dev
 - defaults
 
 packages: &pkgs
```

Comment:
- Before merging this PR, let's wait for HoloViews 1.19.0 to be released, and then remove `pyviz/label/dev` and update the lock file
- Please add/update the lower pins. After doing so, you'll have to re-lock the project, so I recommend you do it at together with the previous item. This step is described in the checklist:

> Add/update the lower pin of all other dependencies (e.g. `hvplot` to `hvplot>=0.9.2`, `hvplot>=0.8` to `hvplot>=0.9.2`). Usually, the new/updated lower pin of a dependency will be the version resolved after `anaconda prepare` has been run. Execute `!conda list` in a notebook, or `anaconda run conda list` in the terminal, to display the version of each dependency installed in the environment. Adjusting the lower pin helps ensure that the locks produced for each platform (linux-64, win-64, osx-64, osx-arm64) rely on the tested dependencies and not on some older versions.

Reply:
OK. We wait!